src/version.ts

```diff
@@ -1 +1 @@
-export const LIB_VERSION = "3.0.1";
+export const LIB_VERSION = "3.0.2"
```

Comment:
typically version numbers aren't updated in PRs, only when releasing a new version